src/openrct2/actions/LandSetHeightAction.hpp

```diff
@@ -105,6 +105,15 @@ DEFINE_GAME_ACTION(LandSetHeightAction, GAME_COMMAND_SET_LAND_HEIGHT, GameAction
         if (surfaceElement == nullptr)
             return std::make_unique<GameActionResult>(GA_ERROR::UNKNOWN, STR_NONE);
 
+        // We need to check if there is _currently_ a level crossing on the tile. For that, we need the old height,
+        // so we can't use the _height variable.
```

Comment:
This is very minor, but this would probably flow better with each sentence on their own line.